Small fixes and improvements

osrecio opened the pull request:
Add FileSystem to manage Files
Fix Construct Controllers Path
Delete Object Manager
Fix PHPCS and PHPMD Problems

@wiardvanrij Check all functionality before Merge. I checked all I know but you have more know-how about your module than me.

Commit:
Add FileSystem to manage Files
Fix Construct Controllers Path
Delete Object Manager
Fix PHPCS and PHPMD Problems

wiardvanrij commented:
Thank you, I will test & check this soon :)

wiardvanrij commented on Console/Command/Clear.php:
```diff
@@ -98,36 +96,51 @@ private function clearRequest()
-
-        if (file_exists($file)) {
-            unlink($file);
+        $fileName = 'allow-opcache.flush';
```
can you make a const of this?

osrecio replied:
Yes I will change it for constant

JKetelaar commented on Console/Command/Clear.php:
```diff
@@ -5,63 +5,61 @@
-     * @return \Symfony\Component\Console\Output\OutputInterface|null
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     * @return int|null|void
```
It returns nothing (void)

osrecio replied:
I will apply 
```
/**
     * {@inheritdoc}
     */
```

JKetelaar commented on Console/Command/Clear.php:
```diff
@@ -5,63 +5,61 @@
+    private $storeManager;
 
     /**
      * @var \Magento\Framework\Filesystem
```
```
* @var Filesystem
```

osrecio replied:
I will change it 👍

Commit:
Add Constant for Filename
Change PHPDocBlock for Execute Function
Use alias class for PHPDockBlock Variable

osrecio commented:
@JKetelaar @wiardvanrij Thanks for your suggestions. I have fixed what you told me.

wiardvanrij commented:
@osrecio thank you very much 👍
